junit-platform-gradle-plugin/src/main/groovy/org/junit/platform/gradle/plugin/JUnitPlatformExtension.groovy

```diff
@@ -37,11 +44,18 @@ class JUnitPlatformExtension {
 	/**
 	 * The directory for the test report files.
 	 *
-	 * <p>Defaults to {@code file('build/test-results/junit-platform')}.
+	 * <p>Defaults to {@code file("$buildDir/test-results/junit-platform")}.
 	 */
 	File reportsDir
 
 	/**
+	 * Accepts Object, and will convert to file automatically using {@link org.gradle.api.Project#file(Object)}
+	 */
+	void setReportsDir(Object reportsDir) {
+		this.reportsDir = project.file(reportsDir)
+	}
```

Comment:
This appears to be the source of the failure on Windows.

The input path `"/any"` in the Spock test gets converted to a relative path instead of an absolute path, which is obviously incorrect.